### DISCUSSION TITLE
Keyword 'achievements' not found.

### DISCUSSION THREAD

**Discussion by JiaxuanGood** [obster-y/XJTU-thesis#128] [Q&A]
\thesisachivements[auto]会被黄线标出，编译产生的pdf中也没有文献成果这一部分的内容。
![微信图片_20240718112148](https://github.com/user-attachments/assets/2811fc25-dd02-457e-9f57-5f887e896408)

**Comment by obster-y** [comment]
和提示无关
1. 是否导入了成果数据库
https://github.com/obster-y/XJTU-thesis/blob/c92bb01721af8477df23ada3a3a0dce57f1b0947/main.tex#L119
2. 是否正确编译

**Reply by JiaxuanGood** [reply]
1. 成果数据库是有的。
2. 可以编译，但不确定什么是正确编译。现在是可以看到pdf

**Reply by JiaxuanGood** [reply]
而且使用\thesisachivements是可以成功看到成果章节的，使用\thesisachivements[auto]不可以。已经有achievement.bib文件和\addachivementresource{References/achievement}命令了。

**Reply by obster-y** [reply]
https://github.com/obster-y/XJTU-thesis/blob/c92bb01721af8477df23ada3a3a0dce57f1b0947/XJTU-thesis.cls#L1664-L1673

回去看了一下，当初的思路和 `main.tex` 里和 Wiki 里说明的使用方法一致，也就是使用 `auto` 参数会自动读取数据库，否则读取 .tex。刚才试了，这个参数的添加与否是能够正确影响结果的。也就是说模板目前看来没有问题，而是你的使用上出了问题

建议提供 MWE，或者自己尝试逐行注释一类的方法看看问题所在
